deploy/L1/migrator.deploy.ts

```diff
@@ -0,0 +1,30 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/dist/types';
+import {ARBITRUM_NETWORK} from '../constants';
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+
+  const {deployer} = await getNamedAccounts();
+
+  const bondingManager = '0x595ab11a0bffbca8134d2105bcf985e85732af5c';
```

Comment:
Fine for now, but separately we should look into parameterizing this so the deployment scripts can be more flexible i.e. make it easier to run this script with any addresses swapped in.